source/solution_deploy/lib/appregistry/applyAppRegistry.ts

```diff
@@ -63,7 +63,12 @@ export class AppRegister {
     const map = this.createMap(stack);
 
     const application = new Application(stack, 'AppRegistry', {
-      applicationName: Fn.join('-', [map.findInMap('Data', 'AppRegistryApplicationName'), Aws.REGION, Aws.ACCOUNT_ID]),
+      applicationName: Fn.join('-', [
+        map.findInMap('Data', 'AppRegistryApplicationName'),
+        Aws.STACK_NAME,
+        Aws.REGION,
+        Aws.ACCOUNT_ID,
+      ]),
```

Comment:
on line 88 of [the cdk template](source/solution_deploy/bin/solution_deploy.ts), the app name is set as "automated-security-response-on-aws" which is now being appended to the Stack_name, Region, and Account_Id -- There is (or at least was) a 130 character limit for the concatenation of "AWS_AppRegistry_Application-" and the application name. 

In theory we are currently consuming 86 characters of that upper limit and leaving only 44 characters for the stack name before deployment will supposedly fail. It may be worth testing if the stack does indeed fail to deploy for a stack name of 45 characters in length. If so, should we potentially append ASR instead of the full "automated-security-response-on-aws" to push the 44 character limit up to 75 for the customer stack name?

Reply:
The app name limit got increased to 300 chars. Stack names are limited to 128 chars. This is fine.